acceptance/testdata/restorer/Dockerfile

```diff
@@ -10,8 +10,10 @@ COPY ./container/ /
 
 # turn /to_cache/<buildpack> directories into cache tarballs
 # these are referenced by sha in /cache/committed/io.buildpacks.lifecycle.cache.metadata
-RUN tar cvf /cache/committed/sha256:b89860e2f9c62e6b5d66d3ce019e18cdabae30273c25150b7f20a82f7a70e494.tar -C /to_cache/cacher_buildpack layers
+RUN tar cvf /cache/committed/sha256:2d9c9c638d5c4f0df067eeae7b9c99ad05776a89d19ab863c28850a91e5f2944.tar -C /to_cache/cacher_buildpack layers
```

Comment:
We realized that creating the tar files in this way (at test time) means the digest will be different every time, as the timestamps within the tar will change depending on when the files were uploaded. Rather than trying to improve our tar command, I opted to just pre-create the tar fixtures.